client/src/components/Navbar.js

```diff
@@ -13,7 +13,7 @@ const Footer = styled.footer`
   display: flex;
   justify-content: space-evenly;
   align-items: center;
-  position: fixed;
+  position: sticky;
```

Comment:
sticky means, that it stays at this position when the actual dom-element enters the viewpoint on the screen to, if you see bottom:0 it have to touch that, and that it will stay there, probably absolut would fix that
```suggestion
  position: absolut;
```

Reply:
Good point! Just a small typo: ` position: absolute;`

Reply:
this only works when my container has 100vh, but then my header isn't sticky anymore :(